src/actionModifiers/ImageExport/ImageExportModifier.js

```diff
@@ -0,0 +1,37 @@
+import SessionService from 'services/session/SessionService';
+import { formatNumberSequence } from 'ommUtils/strings';
+
+function imageExportModifier(openmct) {
+  const PNGImageExportAction = openmct.actions._allActions['export-as-png'];
+  const JPGImageExportAction = openmct.actions._allActions['export-as-jpg'];
+  const imageExportActions = [PNGImageExportAction, JPGImageExportAction].filter(Boolean);
```

Comment:
is this a safeguard? i think we'd just want to know if the actions were broken

Reply:
dunzo